kubernetes/clusters/dev/values.yaml

```diff
@@ -0,0 +1,414 @@
+zookeeper:
```

Comment:
Can you have a comment in ` kubernetes/clusters/min/values.yaml` saying that it uses the default values?